### PR TITLE
Woodmart Sticky AddToCart Support

### PR DESCRIPTION
**Pull request by angbpy**

### CODE REVIEW

File: btn-1cc-checkout.js [hunk 1]
```diff
@@ -43,6 +43,56 @@ function btnCheckout(){
     });
   }
 
+  // For attaching event listener to Woodmart's sticky add-to-cart
+  document.addEventListener('scroll',(e)=>{
+     
+    var stickyBtn=document.querySelectorAll('#btn-1cc-pdp')[1];
+
+    if (stickyBtn != null) {
+        stickyBtn.onclick = function() {
```
Comment (ChetanGN) [inline btn-1cc-checkout.js:52]: Can we make it a common function and invoke it in both places? @angbpy

Comment (angbpy) [reply btn-1cc-checkout.js]: @ChetanGN Done!

**Comment by ChetanGN** [comment]
@angbpy Quantity is not updating through the sticky header. Please fix it.

**Review by poojarazor — APPROVED** [review]
LGTM

File: btn-1cc-checkout.js [hunk 2]
```diff
@@ -43,47 +43,153 @@ function btnCheckout(){
     });
   }
 
+  // For attaching event listener to Woodmart's sticky add-to-cart
+  document.addEventListener('scroll',(e)=>{
+     
+    var stickyBtn=document.querySelectorAll('#btn-1cc-pdp')[1];
```
Comment (ChetanGN) [inline btn-1cc-checkout.js:49]: @angbpy This var will be null for all other pages except the product display page. Please handle this case

**Pull request merged**